container/lxd/initialisation_linux.go

```diff
@@ -155,6 +154,11 @@ var configureLXDBridge = func() error {
 		if err != nil {
 			return errors.Trace(err)
 		}
+		// If there are no suitable bridged NICs in the profile,
+		// ensure the bridge is set up and create one.
+		if err := server.verifyNICsWithAPI(getProfileNICs(profile)); err == nil {
```

Comment:
My patch, so my fault :)

This can simply be `if server.verifyNICsWithAPI(getProfileNICs(profile)) == nil`